### PR TITLE
Added Flask Tutorial under Python Category

### PR DESCRIPTION
**Pull request by Lonercode**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -747,7 +747,7 @@ To add a resource or contribute, please check out our contribution guide [here](
 - [TheAlgorithms/Python: All Algorithms implemented in Python](https://github.com/TheAlgorithms/Python)
 - [30SecondsOfCode Python Snippets](https://www.30secondsofcode.org/python/p/1)
 - [Selenium with Python Tutorial](https://www.javatpoint.com/selenium-python#Selenium-with-python)
-
+- [Python Flask Tutorial](https://youtube.com/playlist?list=PLzMcBGfZo4-n4vJJybUVV3Un_NFS5EOgX)
```
Comment (shahednasser) [inline README.md:750]: Add a new line below this

**Review by shahednasser — CHANGES_REQUESTED** [review]
Hello @Lonercode thank you for your contribution. I added a comment below

File: README.md (hunk 1 above)
Comment (Lonercode) [reply README.md:750]: Alright.

**Comment by shahednasser** [comment]
Great thanks!

**Pull request merged**